src/profile-logic/import/chrome.js

```diff
@@ -529,6 +530,7 @@ async function processTracingEvents(
       threadInfo.lastSeenTime =
         (profileEvent.args.data.cpuProfile.startTime: any) / 1000;
       profileChunks = [profileEvent];
+      lineNumberAdjustment = +1;
```

Comment:
I just checked with a normal chrome profile (compared an import in chrome vs an import in the profiler), and I believe that the same adjustment needs to be done for other chrome profiles too. I'll do the change and merge.